common/gtk-3.0/3.18/render-assets.sh

```diff
@@ -1,34 +1,12 @@
-#! /bin/bash
+#!/bin/bash
+set -ueo pipefail
 
-INKSCAPE="/usr/bin/inkscape"
```

Comment:
It would possibly better to leave the absolute path as it is and maybe enable environment variable support.

Reply:
what makes you think what probability of inkscape being installed in, let's say, /usr/local/ is lower than missing `which` utility?